clients/python/skillkit/__init__.py

```
@@ -0,0 +1,12 @@
+from skillkit.client import SkillKitClient
+from skillkit.models import Skill, SearchResponse, HealthResponse, CacheStats, Category
+
+__all__ = [
+    "SkillKitClient",
+    "Skill",
+    "SearchResponse",
+    "HealthResponse",
+    "CacheStats",
+    "Category",
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)